src/typings/sharify.d.ts

```diff
@@ -89,6 +90,7 @@ declare module "sharify" {
       SUBMISSION_ARTIST_NAME: string
       TARGET_CAMPAIGN_URL: string
       TRACK_PAGELOAD_PATHS: string
+      THIRD_PARTIES_DISABLED: string
```

Comment:
Why not `boolean`?

Reply:
No reason, the few "boolean-ish" values I looked at were strings, I see there are some booleans so I will update it.